students/Snayxts/module-2-html/exercises/zdk09/About.html

```diff
@@ -0,0 +1,92 @@
+            <path d="M202.551 60.1055C201.48 60.1055 200.371 60.0356 199.223 59.896C198.09 59.7719 197.059 59.6245 196.128 59.4538V55.5207C197.912 55.2415 199.51 54.9389 200.922 54.6131C202.349 54.3028 203.622 53.8839 204.739 53.3564C205.871 52.8288 206.88 52.1384 207.764 51.2851C208.648 50.4473 209.447 49.369 210.161 48.0502H206.763L198.478 26.6393C198.354 26.2979 198.176 26.0342 197.943 25.848C197.726 25.6618 197.447 25.5144 197.105 25.4058L195.336 24.8473V21.5658L210.836 21.5891V24.9404L207.392 25.6851L212.721 41.3476L217.725 25.6618L214.42 24.9171V21.5658H225.009V24.8473L223.24 25.4058C222.868 25.5144 222.581 25.6618 222.379 25.848C222.178 26.0187 222.015 26.2824 221.891 26.6393L214.187 48.4691C212.807 52.3634 211.201 55.2725 209.37 57.1964C207.555 59.1358 205.282 60.1055 202.551 60.1055Z" fill="black"/>
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M37.4822 28.4545C36.784 30.2698 36.4349 32.473 36.4349 35.064C36.4349 39.6875 37.5908 43.1086 39.9026 45.3273C42.2298 47.5459 45.4182 48.6553 49.4677 48.6553C51.4226 48.6553 53.2378 48.407 54.9135 47.9105C56.6046 47.4141 58.0631 46.615 59.2888 45.5134C60.5145 44.3964 61.4764 42.9302 62.1746 41.1149C62.8883 39.2996 63.2451 37.0887 63.2451 34.4822C63.2451 29.8431 62.0892 26.4143 59.7775 24.1956C57.4657 21.977 54.2774 20.8676 50.2124 20.8676C48.273 20.8676 46.4577 21.1236 44.7666 21.6356C43.0754 22.1321 41.617 22.9389 40.3913 24.056C39.1656 25.1731 38.1959 26.6393 37.4822 28.4545ZM53.6427 36.9132C55.2885 33.3838 56.1674 30.7464 56.2794 29.0011C56.4054 27.2624 55.695 26.0324 54.1483 25.3111C52.8687 24.7144 51.594 24.9332 50.3242 25.9674C49.0751 26.9942 47.6539 29.216 46.0605 32.6329C44.4147 36.1624 43.5288 38.7964 43.4028 40.5352C43.2908 42.2805 44.0082 43.5138 45.555 44.235C46.8346 44.8317 48.1022 44.6097 49.3579 43.5689C50.6277 42.5347 52.0559 40.3161 53.6427 36.9132Z" fill="black"/>
+        </svg>
+
+        <h1>What we do and why we do it</h1>
+
+        <nav>
+            <a href="index.html">Home</a><br>
+            <a href="products.html">Product</a><br> 
+            <a href="#tct">The Company team</a><br>
+            <a href="#gitt">Get in touch</a>
+        </nav>
+    </header>
+    <br>
+
+    <main>
+        <article>
+            <figure>
+                <img src="slike/billy-corgan.jpg" alt="billy-corgan picture">
+                <figcaption>-Billy Corgan, CEO</figcaption>
+            </figure>
+               
+            <header><h2>"Ipsa netus consequatur blanditiis nostrum pretium phasellus sollicitudin, quae ad aut taciti, inventore purus, natus hymenaeos totam labore dictumst harum explicabo, pede sint? Molestiae! Numquam aut. Mus mollit distinctio? Imperdiet, fugit. Similique conubia torquent. Consequatur parturient, consequuntur omnis quia minima quam hymenaeos, taciti distinctio, expedita quos vestibulum exercitation." - Billy Corgan, CEO</h2>
+            </header>
+            
+            <section>

```

Comment:
Osobno smatram da nije potrebno razdvajati članak na sekcije jer sav sadržaj pripada istoj cjelini. Jedino odvajanje koje se po meni treba dogoditi je s novom cjelinom "The Company team".